### DISCUSSION TITLE
To make it more convenient to pull from local Git tree from folder

### DISCUSSION THREAD

**Discussion by GermanAizek** [gitextensions/gitextensions#11604] [Q&A]
### Feature description

Hi everyone,
Please add 3rd item "Local repository" ability to select local repositories in explorer folders, now it is very inconvenient to implement it every time you must write in URL `file:\\` and local path

![image](https://github.com/gitextensions/gitextensions/assets/21138600/b15a41b9-13aa-4a22-b949-35f1b7de3a96)

### Environment

- Git Extensions 4.2.1.17611
- Build b0c0b284876b63c52c77195ada82a85bfadbb083
- Git 2.42.0.windows.1
- Microsoft Windows NT 10.0.19045.0
- .NET 6.0.26
- DPI 96dpi (no scaling)
- Portable: False
- Microsoft.WindowsDesktop.App Versions

```
    Microsoft.WindowsDesktop.App 6.0.25 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
    Microsoft.WindowsDesktop.App 6.0.26 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
    Microsoft.WindowsDesktop.App 7.0.15 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
    Microsoft.WindowsDesktop.App 8.0.1 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
```

**Comment by RussKie** [comment]
I recommend adding your local folder as a "remote" and then you don't have to specify it manually.
